lib/db.js

```diff
@@ -190,7 +190,6 @@ function net_db(params, callback) {
       logger.warning('Problem contacting DITCH server: ' + err.message + "\n" + util.inspect(err.stack));
       return callback(err);
     });
-    req.end();
```

Comment:
@pb82 @rachael-oregan I must admit I have little-to-zero understanding of the code but having the line `callback(null, {stream: req});` twice (line 195 and line198) in codebase looks weird to me. 
When I removed the line 198 the export didn't work. The error was:
`You cannot pipe after data has been emitted from the response.`

If you are sure the code is ok as is, feel free to ignore this comment :)